AK/Bitmap.h

```diff
@@ -50,6 +50,36 @@ class Bitmap {
         memset(m_data, value ? 0xff : 0x00, size_in_bytes());
     }
 
+    int find_last_unset()
```

Comment:
This function looks more like `find_first_set`, no? You're scanning ahead to find the first bit that's 1.

Reply:
These names made much more sense at the time - I agree that they're backwards. I'm actually not even using these methods in the rest of the patch now, whoops!